README.md

```diff
@@ -46,33 +42,32 @@ TLS information might be generated)
 Waymarked Trails needs a database populated with OSM data to work. You first
 need a data file to import. It's probably easiest to grab an PBF of OSM data
 from [Geofabrik](http://download.geofabrik.de/). Once you have that file put it
-into the waymarked-trails-site directory. Data is imported the first time you
+into the directory `data/import/`. Data is imported the first time you
 run `docker-compose up waymarkedtrails` (see below). This starts the PostgreSQL
 container (downloads it if it not exists) and starts a container that runs the
 necessary scripts to populate the database and then starts the app server. If
 you later on want to re-import data you have to delete the existing database
-first. On the next start of the `waymarkedtrails` container data will be
-imported again. The data file can be customized in two ways. You can set the
-import directory and the data file name from within the compose file by
-adjusting the bind mount as described below and by overriding the environment
-variable `DATAFILE`.
+first: `docker rm waymarkedtrailsdocker_db_1`. On the next start of the
+`waymarkedtrails` container data will be imported again. The data file can be customized in two
+ways.
```

Comment:
Is something missing here? Which ways?

Reply:
Thanks for the review. Yes, this sentence is obsolete. It was about setting the bind mount and the DATAFILE variable. But I moved that down into the config paragraph.